java7/lib/src/test/java/com/truelayer/signing/UsageTest.java

```diff
@@ -1,5 +1,6 @@
 package com.truelayer.signing;
 
+import org.hamcrest.core.StringContains;
```

Comment:
Is this required? For code linting you could adopt a similar approach to what we do on the BE lib, leveraging spotless. More [here](https://github.com/TrueLayer/truelayer-java#code-linting).
Anyway, I'd keep this as a separate PR if interested